### PR TITLE
Code challenge as described on the png image.

### PR DESCRIPTION
**Pull request by allanwilsonGithub**
First ever github, github desktop, JS pull request. The actual code worked 1st time. Boom!

### CODE REVIEW

File: codeChallenge2.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var amounts = [124, 48, 268];
```
Comment (sulanto) [inline codeChallenge2.js]: Could be `const`

File: codeChallenge2.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+function addTip(sum){
```
Comment (sulanto) [inline codeChallenge2.js]:
```suggestion
const addTip = sum => { 
```

If you wanna be a cool JS-kiddo you define it like this (the functionality is the same)

File: codeChallenge2.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    if (sum > 0 && sum < 50){
+        return sum*1.2;
+    } else if (sum => 50 && sum < 200) {
+        return sum*1.15;   
+    } else if (sum >= 200) {
+        return sum*1.1;
+    } else {
+        return 0
```
Comment (sulanto) [inline codeChallenge2.js]: missing semicolon!!!!

File: codeChallenge2.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    }
+}
+
+for (i in amounts){
+    var withTip = addTip((amounts[i]));
```
Comment (sulanto) [inline codeChallenge2.js]: extra parentheses

File: codeChallenge2.js (hunk 2 above)
Comment (allanwilsonGithub) [reply codeChallenge2.js]: I do wanna be a cool JS-kiddo. So those are arrow functions, new in ES6 (js version 2015) to shorten the syntax and remove the 'this'. Although I don't know yet what 'this' means :P unless it's similar to constructors in python

File: codeChallenge2.js (hunk 3 above)
Comment (allanwilsonGithub) [reply codeChallenge2.js]: well done. You passed!

File: codeChallenge2.js (hunk 4 above)
Comment (allanwilsonGithub) [reply codeChallenge2.js]: note to self. You can add multiple layers of redundant parenthesis to look like a noob

File: codeChallenge2.js (hunk 2 above)
Comment (allanwilsonGithub) [reply codeChallenge2.js]: I just got onto 'this' in today's lecture. It is very like python in the main concept.
var john = {
    firstname: 'John',
    birthYear: 1994,
    family: ['Jane', 'Mark'],
    isMarried: false,
    calcAge: function(){
        this.age = 2018 - this.birthYear;           //this
    }
}

**Pull request merged**